Formula/swift-format.rb

```diff
@@ -13,9 +13,19 @@ class SwiftFormat < Formula
     sha256 "d8f72c33efc125e2904e1bec2c8942cca75d75cf81dcab7fcf08ba124af16170" => :catalina
   end
 
-  depends_on xcode: ["11.4", :build]
+  # The bottles are built on systems with the CLT installed, and do not work
+  # out of the box on Xcode-only systems due to an incorrect sysroot.
+  pour_bottle? do
+    reason "The bottle needs the Xcode CLT to be installed."
+    satisfy { MacOS::CLT.installed? }
+  end
+
+  depends_on xcode: ["12.0", :build]
 
   def install
+    ENV["TOOLCHAINS"] = MacOS::Xcode.toolchain_path
+    ENV["HOMEBREW_SDKROOT"] = MacOS::Xcode.sdk_path
+    ENV["SDKROOT"] = MacOS::Xcode.sdk_path
```

Comment:
I believe these are set by default, at least the `HOMEBREW_*` variables are.

Reply:
Good catch.  Those were from when I was trying to work through the 10.15 failures.  I've removed them.